src/main/resources/db/migration/V3_5__add_original_aktivitetskrav_uuid.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE AKTIVITETSKRAV
+    ADD COLUMN original_aktivitetskrav_uuid UUID;
```

Comment:
Jeg ser for meg at dette vil være referanse til forrige aktivitetskrav i tilfellet. Blir kanskje enklere da om man har flere runder med ny_vurdering :thought_balloon:  I så fall er kanskje `previous_aktivitetskrav_uuid` bedre?

Reply:
Så for meg at alle skulle peke på det originale tilfellet, men det blir kanskje greit om alle bare peker på det forrige😊

Reply:
Tenkte at ny-vurdering-apiet må ta utgangspunkt i det forrige (og sjekke at det er i sluttilstand osv) og da blir det kanskje litt enklere om vi bare bruker uuid fra det. Man klarer vel uansett å spørre seg tilbake til "original-tilfellet" om man skulle ha behov for det.